src/app/components/applications/SSV/RegisterValidatorHome/RegisterValidatorHome.tsx

```diff
@@ -22,46 +47,58 @@ const RegisterValidatorHome = () => {
   const operatorStore: OperatorStore = stores.Operator;
   const validatorStore: ValidatorStore = stores.Validator;
 
-  const preRequisites = [
-      'An active Ethereum validator (deposited to Beacon Chain)',
-      'SSV tokens to cover operational fees',
-  ];
-
   useEffect(() => {
       validatorStore.clearKeyStoreFlowData();
-  });
+  }, []);
+
+  const createValidatorsLaunchpad = () => {
+    navigate(config.routes.SSV.VALIDATOR.CREATE);
+  };
 
   const moveToSelectOperators = () => {
     processStore.setProcess({
       item: null,
       processName: 'register_validator',
-    }, 2);
-    operatorStore.unselectAllOperators();
-    operatorStore.setClusterSize(config.FEATURE.OPERATORS.SELECT_MINIMUM_OPERATORS);
+    }, ProcessType.Validator);
+    operatorStore.unselectAllOperators(); // TODO should later update these from the keyshares file info
+    operatorStore.setClusterSize(config.FEATURE.OPERATORS.SELECT_MINIMUM_OPERATORS); // TODO should later change this from the keyshares file info
```

Comment:
Add these todos to your tasks to not forget

Reply:
@alexstolr please pay attention

Reply:
These are already handled. deleting the comments.